webapp-django/crashstats/signature/views.py

```diff
@@ -439,6 +439,32 @@ def signature_summary(request, params):
         params['_histogram.date'] = ['exploitability']
 
     api = SuperSearchUnredacted()
+
+    product_version_facet = None
+
+    # If there is a version or a product filter, we need to make a separate
+    # query so that we can show all versions and not just the one asked for.
+    if (
+        ('version' in params and params['version'])
+        or ('product' in params and params['product'])
+    ):
+        params_copy = dict(params)
```

Comment:
that makes a shallow copy. Is that OK? I.e. are there no deeper levels that 1 level?